### ISSUE TITLE
读中文的年份的时候有错误

### ISSUE THREAD

**Issue by jonahzheng** [myshell-ai/MeloTTS#85]
比如‘2018年’
生成的语音会念成er qian ling yi shi ba nian，应该是‘liang qian ling yi ba nian’ 或 ‘er ling yi ba nian’
这个要如何修正，重新用语料训练吗？

**Comment by MissingTwins** [comment]
Yes, there are many issues with how years are pronounced, and the percent symbol (%) is not pronounced at all for both Chinese and English. Also, many capitalized abbreviations are not correctly pronounced, such as HBO, CCTV, XBOX, and HIV.

···
from melo.api import TTS

device = 'auto'
speed = 1.0
text = '说中文, ni3 hao3吗, BBC广播, CNN新闻, 美国HBO, NBC转播, CCTV一套, HIV病毒, 100%不合格, 列宁格勒，希特勒，P2P软件, Sony的PSP, 微软的XBOX游戏机, 2018年初，1931年, 轰炸地堡，数了数几只羊'

model_tts = TTS(language=‘ZH’, device=device)
speaker_ids = model_tts.hps.data.spk2id
model_tts.tts_to_file(translated_output_text, speaker_ids['ZH'], 1.wav, speed=speed)
···

[![Demo Audio]](https://github.com/myshell-ai/MeloTTS/assets/146804746/a16e575b-bb66-4091-9afc-1b0261a42454)

**Comment by liuxiong21** [comment]
你的中文预训练模型在哪里下载的？

**Comment by Simon-chai** [comment]
> model_tts.tts_to_file(translated_output_text, speaker_ids['ZH'], 1.wav, speed=speed)
speaker_ids['ZH'] raise error AttributeError: 'HParams' object has no attribute 'ZH'
what have I missed?

**Comment by gurudk** [comment]
中文开源生态真的道阻且长，按理说，这种开放语料就应该通过众包的方式解决，英文的开放语料库多多了

**Comment by smallfish45** [comment]
er qian ling yi shi ba nian、liang qian ling yi ba nian、er ling yi ba nian这种不同念法感觉可以g2p时处理
